src/components/common/Footer/Footer.tsx

```diff
@@ -3,24 +3,29 @@ import { AiOutlineTwitter } from 'react-icons/ai'
 
 const Footer: React.FC = () => {
   return (
-    <footer className='footer bg-neutral p-4 text-neutral-content'>
-      <div className='grid-flow-col items-center'>
-        <Image src='/logo_images.png' width={36} height={36} alt='logo' />
-        <p>Copyright © 2023 - NUTMEG</p>
-      </div>
-      <div className='grid-flow-col gap-4 md:place-self-center md:justify-self-end'>
-        <a
-          className='hover:scale-125'
-          href='https://twitter.com/nutfes_nutmeg?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Eauthor'
-        >
-          <AiOutlineTwitter size='24' />
-        </a>
-        <a className='link-hover link hover:scale-125' href='https://blog.nutmeg.cloud/'>
-          Blog
-        </a>
-        <a className='link-hover link hover:scale-125' href='https://www.nutfes.net/'>
-          HP
-        </a>
+    <footer className='footer bg-neutral p-1 md:p-4 text-neutral-content'>
+      <div className='w-full grid-flow-col items-center'>
+        <Image src='/logo_images.png' width={24} height={24} alt='logo' />
+        <div className='text-xs md:text-base md:place-self-center'>
```

Comment:
```suggestion
        <div className='ml-12 place-self-center text-xs md:text-base'>
```
ml当てたらpcとsp両方真ん中になった